.github/scripts/js/ci.js

```diff
@@ -779,17 +896,63 @@ const findAndRerunWorkflow = async ({ github, context, core, workflow_id }) => {
  */
 module.exports.createReleaseIssueForMilestone = async ({ github, context, core }) => {
   const milestone = context.payload.milestone;
+
+  const found = milestone.title.match(/(v\d+\.\d+\.\d+)/);
+  if (!found) {
+    return core.setFailed(`Milestone '${milestone.title}' not dedicated to release version in form of vX.Y.Z. Ignore creating release issue.'`);
+  }
+  const milestoneVersion = found[1];
+
   // NOTE: non-breaking space after emoji.
-  const issueBody = `:point_right: Use this issue to test milestone [${milestone.title}](${milestone.html_url}) and deploy released tag.
-            :point_right: Use 'e2e/run/' labels to run default e2e test.
-            :point_right: Use 'e2e/use/' labels to run specific e2e test.
-            :point_right: Use 'deploy/' labels to deploy site and documentation.
-            :point_right: Use 'deploy/deckhouse/' labels to deploy to channels after creating tag.`;
+  const issueBody = `:robot: A dedicated issue to run tests and deploy release [${milestoneVersion}](${milestone.html_url}).
+
+---
+
+<details>
+<summary>Release issue commands and options</summary>
+<br />
+
+You can trigger release actions by commenting on this issue:
+
+- \`/deploy/<channel>\` will publish built images into the release channel.
+  - \`channel\` is one of \`alpha | beta | early-access | rock-solid | stable\`
+- \`/suspend/<channel>\` will suspend released version.
+  - \`channel\` is one of \`alpha | beta | early-access | rock-solid | stable\`
+- \`/e2e/run/<provider> git_ref\` will run e2e using provider and an \`install\` image built from git_ref.
+  - \`provider\` is one of \`aws | azure | gcp | yandex-cloud | openstack | vsphere | static\`
+  - \`git_ref\` is a tag or branch: \`vX.Y.Z | test-X.Y.Z* | main | release-X.Y\`
+- \`/e2e/use/cri/<cri_name>\` specifies which CRI to use for e2e test.
+  - \`cri_name\` is one of \`docker | containerd\`
+- \`/e2e/use/k8s/<version>\` specifies which Kubernetes version to use for e2e test.
+  - \`version\` is one of \`1.19 | 1.20 | 1.21 | 1.22\`
```

Comment:
How can we update this doc when we start supporting 1.23, etc. and drop the support of 1.19 and 1.20?

I believe, we should not maintain this place, just add examples.

Reply:
Oh! We can fix it for new issues by generating the list from constants.js.

Reply:
I've added generating for editions, providers, channels, CRIs and versions.